### PR TITLE
GODRIVER-1770 Use chunkSize from files doc for download streams

### PR DESCRIPTION
**Pull request by divjotarora**

### CODE REVIEW

File: mongo/gridfs/bucket.go [hunk 1]
```diff
@@ -381,14 +381,16 @@ func (b *Bucket) openDownloadStream(filter interface{}, opts ...*options.FindOpt
 	}
 
 	if foundFile.Length == 0 {
-		return newDownloadStream(nil, b.chunkSize, &foundFile), nil
+		return newDownloadStream(nil, foundFile.ChunkSize, &foundFile), nil
```
Comment (iwysiu) [inline mongo/gridfs/bucket.go:384]: I assume from this that foundFile will always have a chunkSize?

**Review by iwysiu — APPROVED** [review]
LGTM with question

File: mongo/gridfs/bucket.go (hunk 1 above)
Comment (divjotarora) [reply mongo/gridfs/bucket.go:387]: Yeah, it's a required field. I updated the code to add a new error if it's missing.

**Review by kevinAlbs — APPROVED** [review]
LGTM!

**Pull request merged**